Webex Connect Flows/README.md

```diff
@@ -8,13 +8,51 @@ are versioned and Webex Contact Center deployment **region agnostic**.
 - Digital flows contain various nodes that enable you to configure success and error paths for the execution of customer interactions.
 - For Contact Center to function, some nodes need to be arranged in specific order, hence templates and samples are provided to depict the correct usage of the flows.
 
+## 3.1 Workflows
+In 3.1 version of workflows, we have the simplified flows enhanced with Security scan results being passed on to the backend to further protect your contact centre against malware content coming in via attachments in messages.
```

Comment:
> enhanced with Security scan results being passed on to the backend to further protect your contact centre against malware content coming in via attachments in messages.

protecting contact center against malware content is by default available to all flows old, 3.0 & 3.1 after malware GA. Can we rephrase the description to indicate like the flow builder and respective backend services are enhanced to support scanning malware content in attachments.

Reply:
Sure, let us discuss once on how we want to reword it and I will incorporate the same.